### PR TITLE
Add "G.O.D.-Eyes Phantom Dragon (Manga)

### PR DESCRIPTION
**Pull request by 42Rundas**
https://yugipedia.com/wiki/G.O.D.-Eyes_Phantom_Dragon_(manga)

Supervising Staff Members would be senpaizuri3 and pyrQ.

### CODE REVIEW

File: c200000005.lua [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+	if c==nil then return true end
+	local tp=c:GetControler()
+	local g=Duel.GetFieldGroup(tp,LOCATION_MZONE,0)
+	local rg=Duel.GetReleaseGroup(tp)
+	return (#g>0 or #rg>0) and g:FilterCount(Card.IsReleasable,nil)==#g and #g>0 and g:IsExists(s.pfilter,1,nil)
+		and g:FilterCount(s.cfilter,nil)+Duel.GetLocationCount(tp,LOCATION_MZONE)>0
+end
+
+function s.matfilter(c,tp)
+	return c:IsRace(RACE_DRAGON) and c:IsType(TYPE_PENDULUM) and Duel.GetReleaseGroup(tp):IsContains(c)
```
Comment (edo9300) [inline c200000005.lua:92]: here tp is not being passed, also the ``Duel.GetReleaseGroup(tp):IsContains(c)`` is redoundant as you're already iterating ReleaseGroup

Comment (42Rundas) [reply c200000005.lua]: Ok, Changed the filter as well as the spop.

**Comment by 42Rundas** [comment]
The cdb is now uploaded in the correct repo. Also I updated the script.

**Comment by kevinlul** [comment]
You've invalidated this PR by making it about something else. Please use separate branches to separate concerns.

**Pull request closed without merging**